core/src/main/resources/io/questdb/site/conf/server.conf

```diff
@@ -837,6 +837,9 @@ cairo.sql.copy.root=import
 # Maximum number of rows inserted by the refresh job in a single transaction.
 #cairo.mat.view.insert.as.select.batch.size=1000000
 
+# Maximum number of time intervals from WAL data transactions cached per materialized view.
+#cairo.mat.view.max.refresh.intervals=100
+
```

Comment:
nit: please add the second property

Reply:
Documented in aa9b10a34b22da6db3d87a3f5afb0fb5e0aab275